src/main/java/duke/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.UI;
+import duke.DukeException;
+import duke.Task;
+import duke.Deadline;
+import duke.Parser;
+
+/**
+ * AddDeadlineCommand class to represent an instruction to add a new Deadline task to the TaskList.
+ *
+ * @author Sheryl Kong (A0240686Y)
+ */
+
+public class AddDeadlineCommand extends Command { //Creating a duke.Deadline duke.Task and adding to the taskList
+    String description;
+    boolean isDone;
+    String date;
+
+    public AddDeadlineCommand(String description, boolean isDone, String date) {
+        this.description = description;
+        this.isDone = isDone;
+        this.date = date;
+    }
+
+    @Override
+    public void execute(TaskList taskList, Storage storage) throws DukeException {
+        if (!storage.checkIsLoadingFile()) { this.date = Parser.parseDate(date); }
```

Comment:
Perhaps you could use Egyptian style braces? 
```suggestion
        if (!storage.checkIsLoadingFile()) {
            this.date = Parser.parseDate(date);
        }
```

Same for line 34